.packages

```diff
@@ -0,0 +1,86 @@
+# This file is deprecated. Tools should instead consume 
```

Comment:
Add this file to `.gitignore` and remove it from the commits

Reply:
it's strange because it is already in .gitignore. But I just deleted it